CustomerOrganizer/CustomerOrganizer/MainWindow.py

```diff
@@ -3,26 +3,25 @@
 import FileIO
 from FileIO import *
 
-class MainWindow(tkinter.Tk):
+class Main_Window(tkinter.Tk):
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.toolBar()
+        self.customer = FileIO()
+        self.tool_bar()
 
-        self.customer = FileIO.FileIO()
-
-    def donothing():
+    def do_nothing():
         '''do nothing at all'''
 
-    def toolBar(self):
+    def tool_bar(self):
         menubar = Menu(self)
 
         filemenu = Menu(menubar, tearoff = 0)
-        filemenu.add_command(label = "New Customer", command = self.donothing)
-        filemenu.add_command(label = "Open Customer", command = self.donothing)
+        filemenu.add_command(label = "New Customer", command = self.do_nothing)
+        filemenu.add_command(label = "Open Customer", command = self.do_nothing)
         filemenu.add_command(label = "Save Customer", command = self.customer.saveFile)
```

Comment:
Please rename saveFile to save_file. Also do the same in FileIO.py

Reply:
gotcha